app/herbivore/herbivore.py

```diff
@@ -0,0 +1,9 @@
+from ..animal.animal import Animal
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        if self.hidden:
+            self.hidden = False
+        else:
+            self.hidden = True
```

Comment:
read it [https://github.com/mate-academy/py-herbivores-and-carnivores/blob/master/checklist.md](url)
![image](https://github.com/mate-academy/py-herbivores-and-carnivores/assets/93856423/e8ff58ff-a375-4a8d-9fce-db6f764f0280)